pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-gb.md

```diff
@@ -0,0 +1,444 @@
+
+Click on the `Create a database instance`{.action} button. (`Create a service`{.action} if your project already contains databases.)
+
+#### Step 1: Select PostgreSQL database
+
+Click on the PostgreSQL database and then select the version to install from the drop-down menu. Click `Next`{.action} to continue.
+
+![Choose PostgreSQL database](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql01.png){.thumbnail}
+
+Click `Next`{.action} to continue.
+
+#### Step 2: Select a solution
+
+In this step, choose an appropriate service plan. You will be able to upgrade the plan after creation.
+
+![Choose plan](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql02.png){.thumbnail}
+
+Please visit the [capabilities page](https://docs.ovh.com/gb/en/publiccloud/databases/postgresql/capabilities/) for detailed information on each plan's properties.
+
+Click `Next`{.action} to continue.
+
+#### Step 3: Select a location
+
+Choose the geographical region of the data center in which your database will be hosted.
+
+![Choose region](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql03.png){.thumbnail}
+
+Click `Next`{.action} to continue.
+
+#### Step 4: Configure database nodes
+
+You can choose the node model in this step. The initial and maximum numbers of nodes depend on the chosen solution in step 2.
+
+![Order nodes](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql04.png){.thumbnail}
+
+Please visit the [capabilities page](https://docs.ovh.com/gb/en/publiccloud/databases/postgresql/capabilities/) for detailed information on the hardware resources and other properties of the MySQL database installation.
+
+Take note of the pricing information and click `Next`{.action} to continue.
+
+#### Step 5: Configure your options
+
+You can name your database in this step.
+
+![Configure options](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql05.png){.thumbnail}
+
+Click `Next`{.action} to continue.
+
+#### Step 6: Summary and confirmation
+
+The last section will show a summary of your order as well as the API equivalent of creating this database instance with the [OVHcloud API](https://docs.ovh.com/gb/en/api/first-steps-with-ovh-api/).
+
+![Confirm order](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql06.png){.thumbnail}
+
+Click `Create a database service`{.action} to continue.
+
+Within a few minutes your new database service will be deployed. Messages in the OVHcloud Control Panel will inform you when the database is ready to use.
+
+Your database is ready when cluster status is "Ready", node status is green, and number of users is set to "1 user".
+
+![Green statuses on Database Control Panel](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql08.png){.thumbnail}
+
+### Authorise your IP address
```

Comment:
"Authorize your IP address"

Reply:
debate is open :)

Reply:
The debate is closed, "authoriZe" won :)